### DISCUSSION TITLE
Communicating the intent of "Top"

### DISCUSSION THREAD

**Discussion by lovelydinosaur** [community/community#3706] [Discussions]
It's not immediately obvious to me what precise intent/function the "Top" filter provides.

For example, filtering on this repo we get this...

<img width="1245" alt="Screenshot 2021-04-06 at 12 49 03" src="https://user-images.githubusercontent.com/647359/113706581-c22f9c80-96d6-11eb-91c9-4bd53fa48556.png">

Issues that spring to my mind as a user are:

* I'd expected this button to filter "most upvotes first", but it doesn't *look* like that's what it actually is.
* The button *could* be visually indicated as in a "pressed" state, but isn't.

**Comment by jen-k** [comment]
Hey @tomchristie 👋  We calculate the "Top" discussions using a combination of engagement types including upvotes and comments. I agree that this might seem a bit confusing, and I'm happy to pass your comments on to our engineering team to see how the feature might be improved. Thanks so much for reaching out!

**Reply by jasonvarga** [reply]
We plan to use Discussions as a place to house feature requests.
Previously we used issues and told our users to give 👍 reactions to them, so we could see the most reacted issues.

Now in Discussions there doesn't seem to be a consistent way to sort by most reacted. It's this magical "top" metric. Should we tell our users to upvote discussions? React to them? At the moment there's no way to sort by upvotes or reactions alone.

**Reply by rijkvanzanten** [reply]
Same here. We rely on Discussions for feature requests, but without a way to sort by actual upvotes, there's no way to know what the most requested feature is. It would be great if there was a separate "sort:upvotes", similar to the `sort:reactions-+1-desc` you'd find on regular Issues

**Reply by rebelchris** [reply]
+1 for this feature

**Reply by jaywalker21** [reply]
@jen-k Any discussions with the dev/product team around the same? We were also planning to leverage discussions for the Feature Request and upvote tracking. Will really make life much simpler for everyone.

**Comment by friday** [comment]
I'm also confused about the "New" button next to Top. I pressed, thinking it would then switch to show the overall top results, but it doesn't. It is the default view. People have to click twice to select another view, and only then they can click it to reset back to the default view.

I think it should be easy to get a view where the discussions are sorted only by upvotes. I would want to make it the default for sum Topics. But now it's not even possible.
